### ISSUE TITLE
Contant crashing well using leveling 

### ISSUE THREAD

**Issue by Bxmnx** [Lailloken/Lailloken-UI#324]
I'm getting an error stating Invalid or Non existant window owner when chaning zones with the leveling tracker on.
![AutoHotkey_H7gJeQmZIT](https://github.com/Lailloken/Lailloken-UI/assets/91206892/43c4b82a-cca2-45c5-9618-780aaddf171e)
Here are the settings for leveling tracker
![AutoHotkey_86rzbtp35Y](https://github.com/Lailloken/Lailloken-UI/assets/91206892/d2999bea-a2e4-43a9-9313-0f16e78681ab)

This actually happens on my gaming PC and my laptop. Both are running windows 10

**Comment by Lailloken** [comment]
That's the first time I see this kind of issue. The guide-overlay is a layered window, and it looks as if the script cannot "find" the first layer in order to place the second one on it. This seems to be a compatibility issue and not an inherent bug.

Here are some things you could try to troubleshoot:
- in the `general` tab of the settings menu, there's a button to restart the script as administrator (maybe Windows has an issue with the script being stored on C:\ and is blocking its access)

- try moving the folder to a secondary drive and launch it from there
- reinstall AutoHotkey

Are you running a tweaked Windows install, or any software that may interact with or modify windows that are open on your screen?

**Comment by null77** [comment]
I tried without SetControlDelay/SetWinDelay (which would default to 20ms delay) and with a custom 200ms delay, and still was getting crashes, so it seems like unfortunately that won't fix the issue.

**Comment by Lailloken** [comment]
I'll see if I can implement some sort of safe-mode before league-launch, a kind of fallback that uses simple windows instead of composited ones.

**Comment by Lailloken** [comment]
@Bxmnx Have things improved on your end? Did moving the folder make a difference?

**Comment by null77** [comment]
Just as a data point - I used the UI for my whole league start leveling and it only crashed once. Was great!

**Comment by metzbot** [comment]
Hello, chiming in with more feedback on this issue. Mostly happened during zone transitions, rarely when I tried interacting with the widget (clicking the current step forward or back). Oddly enough, it was most consistently reproduceable when loading into a lab run.

![image](https://github.com/Lailloken/Lailloken-UI/assets/98874405/623b7e3c-6cca-4d55-84eb-cfacce0f6b74)

**Comment by Lailloken** [comment]
[Here](https://github.com/Lailloken/Lailloken-UI/releases/tag/v1.51.3-test) is a test-version of v1.51.3 that contains a potential fix for this issue (apparently, it's Windows-related). It will still show as v1.51.2 in the updater in order to show a notification once 1.51.3 is ready for official release. Simply overwrite the files inside your current install with the files inside `ZIP FILE\Lailloken-UI-test-1.51.3-test\`. In case there are problems with this test-version, simply roll back to [v1.51.2](https://github.com/Lailloken/Lailloken-UI/releases/tag/v1.51.2) by doing the same procedure.

Would be cool if the affected people could take five minutes to zoom through acts again after resetting the tracker, then report their initial findings. Or simply use it on the next character.

**Comment by metzbot** [comment]
Hi, I'm running the test version. I'm still seeing errant behavior. It seems like there is some strange window detection happening (for reference, I am using windowed borderless). Sometimes when clicking on the leveling tracker (or any LUI thing for that matter, Path of Exile will lose current window focus. I'm also noticing strange popups from other LUI elements on the screen and I think it might be related. Seen here:

![image](https://github.com/Lailloken/Lailloken-UI/assets/98874405/858f7297-a3b1-4d30-bd36-b5eb7a8f4c6b)

**Comment by Lailloken** [comment]
@metzbot Thanks for your feedback. It looks like this experimental/band-aid fix makes things somewhat worse. I already have an alternative fix, but that one requires much more work because I have to restructure how the tool handles its windows/overlays.

**Comment by metzbot** [comment]
> @metzbot Thanks for your feedback. It looks like this experimental/band-aid fix makes things somewhat worse. I already have an alternative fix, but that one requires much more work because I have to restructure how the tool handles its windows/overlays.

Thanks for your hard work! Very grateful for this tool. It has been flawless for a few leagues now.

**Comment by Lailloken** [comment]
Let's try again: [v1.51.3 test-release](https://github.com/Lailloken/Lailloken-UI/releases/tag/v1.51.3-test). Use this to update your install to a test-version of v1.51.3.
You can test it while leveling your next character, or now if you also use any other features besides the leveling tracker. If it breaks anything, simply roll back to v1.51.2 (and please report your findings).

**Comment by Fomok** [comment]
after update to v1.51.3 test-release
![image](https://github.com/Lailloken/Lailloken-UI/assets/60652038/a4caef03-5f21-43c4-9572-1f5ff81a1acc)

**Comment by Lailloken** [comment]
@Fomok Did that error occur when transitioning between areas, or was it somewhere else?

**Comment by Fomok** [comment]
yeah same as the first error but more often i think, less flustrating tho cuz never lags out the whole ui with 1 click so i was just restarting the script on taskbar

**Comment by Lailloken** [comment]
Potentially fixed in v1.52.0
Closing this thread for now since the root issue should be fixed. Any emerging issues can be reported in a new thread.
